src/main/webapp/scripts/session-script.js

```diff
@@ -27,6 +37,26 @@ const URL_PARAM_KEY = {
   SESSION_ID: 'session-id'
 };
 
+/**
+ * An array of who is currently in the session.
+ * @type {Array}
+ */
+let currentAttendees = [];
```

Comment:
how is this updated?

Reply:
Line 178 changes the currentAttendees list to be the updated array passed in